addons/dragging/XEH_postInit.sqf

```diff
@@ -54,6 +59,29 @@ if (isNil QGVAR(maxWeightCarryRun)) then {
 // Handle waking up dragged unit and falling unconscious while dragging
 ["ace_unconscious", LINKFUNC(handleUnconscious)] call CBA_fnc_addEventHandler;
 
+// Handle local effect commands for clones
+[QGVAR(cloneCreated), {
+    params ["_unit", "_clone"];
+
+    _clone setFace face _unit;
+    _clone setMimic "unconscious";
+}] call CBA_fnc_addEventHandler;
+
+[QGVAR(moveCorpse), {
+    params ["_corpse", "_dir", "_pos"];
+
+    // Set direction before position
+    _corpse setDir _dir;
+
+    // Bring corpse back to clone's position
+    _corpse setPosATL _pos;
+
+    // Sync the corpse
+    [QEGVAR(common,awake), [_corpse, true]] call CBA_fnc_globalEvent;
+    [QEGVAR(common,awake), [_corpse, false]] call CBA_fnc_globalEvent;
+    [QEGVAR(common,awake), [_corpse, true]] call CBA_fnc_globalEvent;
```

Comment:
This last is not necessary, but it makes the corpse ragdoll, which can be both good and bad: It looks better, but it might cause collisions.